gensim/models/phrases.py

```diff
@@ -425,6 +396,10 @@ def learn_vocab(sentences, max_vocab_size, delimiter=b'_', progress_per=10000,
                     "PROGRESS: at sentence #%i, processed %i words and %i word types",
                     sentence_no, total_words, len(vocab),
                 )
+            
+            if doc2vec:
```

Comment:
What for? Please show several code pieces that demonstrate your idea (why this is needed)